pkg/sidecar/identity/member.go

```diff
@@ -88,8 +92,12 @@ func (m *Member) GetSeed(ctx context.Context, coreClient v1.CoreV1Interface) (st
 
 	svc, err := coreClient.Services(m.Namespace).Get(ctx, pod.Name, metav1.GetOptions{})
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 
-	return svc.Spec.ClusterIP, nil
+	res := make([]string, 0, len(externalSeeds)+1)
+	copy(res, externalSeeds)
```

Comment:
return values need to be processed or explicitly ignored.
(to me, using append looks more idiomatic and consistent but I don't feel strong)

Reply:
done